proto/net.proto

```diff
@@ -198,12 +238,37 @@ message NetworkFormat {
   }
   optional MovesLeftFormat moves_left = 6;
 
+  enum ActivationFunction {
+    DEFAULT = 0;
+    MISH = 1;
+    RELU = 2;
+    NONE = 3;
+    TANH = 4;
+    SIGMOID = 5;
+    SELU = 6;
+    SWISH = 7;
+    RELU_2 = 8;
+    SOFTMAX = 9;
+  }
+
   // Activation used everywhere except head outputs or otherwise specified.
   enum DefaultActivation {
     DEFAULT_ACTIVATION_RELU = 0;
     DEFAULT_ACTIVATION_MISH = 1;
   }
   optional DefaultActivation default_activation = 7;
+
+  enum SmolgenActivation {
+    SMOLGEN_ACTIVATION_INHERIT = 0;
```

Comment:
As discussed in Discord, I'd prefer to reuse the same enum for activations in all places, but I'm fine with separate enums if other people feel strongly about it.

Reply:
Done